plugins/Kaleidoscope-LED-ActiveLayerKeys/README.md

```diff
@@ -0,0 +1,46 @@
+# LED-ActiveLayerKeys
+
+A simple way to light up the keyboard in uniform colors, depending on what layer
+one's on. Unlike [Colormap](Kaleidoscope-Colormap.md), all keys will be the same color. But
+this plugin uses considerably less resources, and is easier to set up as well. A
+perfect solution when one wants to quickly see what layer they're on, with
+minimal resources and time investment.
+
+## Using the plugin
+
+To use the plugin, one needs to include the header, and activate the effect.
+Then, one needs to configure a color map:
+
+```c++
+#include <Kaleidoscope.h>
+#include <Kaleidoscope-LEDControl.h>
+#include <Kaleidoscope-LED-ActiveLayerKeys.h>
+
+KALEIDOSCOPE_INIT_PLUGINS(LEDControl,
+                          LEDActiveLayerKeysEffect);
+
+void setup () {
+  static const cRGB layerColormap[] PROGMEM = {
+    CRGB(0xd0, 0x08, 0x8a),
+    CRGB(0x99, 0x11, 0xb5),
+  };
+
+  Kaleidoscope.setup();
+  LEDActiveLayerKeysEffect.setColormap(layerColormap);
+}
+```
+
+## Plugin properties
+
+The plugin provides the `LEDActiveLayerKeysEffect` object, which has the following
+method:
+
+### `.setColormap(colormap)`
+
+> Sets the colormap to the supplied map. Each element of the map should be a
+> `cRGB` color, and the array must have the same amount of items as there are
```

Comment:
What happens if the user gets the number of layers wrong?   I wonder if it would make sense to set up "default" colors for up to a dozen layers so that users don't need to override the color map unless they want to change them?

Reply:
This behaviour is [exactly the same](https://github.com/keyboardio/Kaleidoscope/blob/master/plugins/Kaleidoscope-LED-ActiveLayerColor/README.md?plain=1#L40-L42) as with the ActiveLayerColor plugin.

It isn't too hard to change that in the [getActiveColor](https://github.com/keyboardio/Kaleidoscope/pull/1355/files#diff-bf4d14835645db8edc064486f531d7939ccfefe3c43c09a3799b842da814b96fR43-R53) method, but perhaps it would be better to have a separate PR to fix that for both plugins?

Reply:
My preference would be to do it right in the new plugin and a PR to fix the old one, but I don't feel super strongly.

Reply:
I *think* it's done, but please check my logic as it comes right from StackOverflow